src/Tribe/Admin/Notices.php

```diff
@@ -137,26 +211,29 @@ public function maybe_display_plus_commerce_notice() {
 		$plus = sprintf(
 			'<a target="_blank" rel="noopener nofollow" href="%s">%s</a>',
 			esc_attr( $plus_link ),
-			esc_html( 'Event Tickets Plus', 'event-tickets' )
+			esc_html__( 'Event Tickets Plus', 'event-tickets' )
 		);
 
-		$plus_commerce_providers = array(
-			esc_html( 'WooCommerce', 'event-tickets' )            => 'woocommerce/woocommerce.php',
-			esc_html( 'Easy Digital Downloads', 'event-tickets' ) => 'easy-digital-downloads/easy-digital-downloads.php',
-		);
+		$plus_commerce_providers = [
+			'woocommerce/woocommerce.php'                       => __( 'WooCommerce', 'event-tickets' ),
+			'easy-digital-downloads/easy-digital-downloads.php' => __( 'Easy Digital Downloads', 'event-tickets' ),
+		];
 
-		foreach ( $plus_commerce_providers as $provider => $path ) {
+		foreach ( $plus_commerce_providers as $path => $provider ) {
 			if ( ! is_plugin_active( $path ) ) {
 				continue;
 			}
 
 			$message = sprintf(
-				__( 'Event Tickets does not support ticket sales via third party ecommerce plugins. If you want to sell tickets with %1$s, please purchase a license for %2$s.' ),
+				__( 'Event Tickets does not support ticket sales via third party ecommerce plugins. If you want to sell tickets with %1$s, please purchase a license for %2$s.', 'event-tickets' ),
```

Comment:
:no_entry_sign: A gettext call containing placeholders was found, but was not accompanied by a "translators:" comment on the line above to clarify the meaning of the placeholders.

<sub>hash: cd86fb304e58c83fce6b46a0457cb5cf</sub>